docs/error-messages/compiler-errors-2/compiler-error-c2860.md

```diff
@@ -1,22 +1,22 @@
 ---
 description: "Learn more about: Compiler Error C2860"
 title: "Compiler Error C2860"
-ms.date: "11/04/2016"
+ms.date: "03/16/2024"
 f1_keywords: ["C2860"]
 helpviewer_keywords: ["C2860"]
-ms.assetid: ccc83553-90ed-4e94-b5e9-38b58ae38e31
 ---
 # Compiler Error C2860
 
-'void' cannot be an argument type, except for '(void)'
+'void' cannot be used as a function parameter except for '(void)'
```

Comment:
Do we prefer using block quotes for the error message, like:
```diff
- 'void' cannot be used as a function parameter except for '(void)'
+ > 'void' cannot be used as a function parameter except for '(void)'
```

How about the usage of `Remarks` and `Example` subheaders?

Reply:
The 'typical' style for these, for no good reason that I know of other than that's what the rest use, is to avoid block quotes and to not add the ## Example or ## Remarks sections. There are some exceptions if you look around, but they are more the exception than the rule. So we may as well stay with the format most of them are in.